our-app/src/components/authenticated/ImmunizationForm.js

```diff
@@ -40,11 +40,10 @@ const ImmunizationForm = (props) => {
         props.submitImmunizationRecord(immunization);
     };
     return (
-        <Card>
+        <Card fluid="xl" style={{ height: '80rem', width: '70rem' }}>
```

Comment:
Can we use bootstrap classes here instead of inline styling? If the answer is no, then I don’t care :)

Reply:
I'm just gonna send it

Reply:
```suggestion
        <Card>
```